app/db/interface.py

```diff
@@ -56,8 +60,29 @@ def update_payment(self, payment_id, payment_replacement=None, payment_attribute
         :param payment_replacement: <dict> a complete payload that describes a new payload which replaces the old one
         :param payment_attributes: <dict> a collection of new payment attribute values that will overwrite old ones
         """
-
-        raise NotImplementedError()
+        payment = self.db.session.query(Payment).get(payment_id)
```

Comment:
this sucker right here (line 78) is what's giving us trouble, though it seems to be a bit more than that.

i was finally able to recreate the db error. this query throws an exception that isn't handled, but it's only throwing it because the tables haven't been built by the db. not sure why so i'm looking into this. just an fyi